config/core/configmaps/default-pingsource.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-ping-webhook
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+  annotations:
+    knative.dev/example-checksum: "6eaeecba"
+data:
+  ping-config: |
+   # dataMaxSize: 4096 # Max number of bytes allowed to be sent for message excluding any base64 decoding.
```

Comment:
I see an example checksum, but no `_example` key in the configmap.

Reply:
Good point. `knative.dev/example-checksum: "6eaeecba"` Is not needed here.